docs/administration-guide/01-managing-databases.md

```diff
@@ -8,12 +8,14 @@ Cool, now you’re in the administration section of Metabase. Next, select **Dat
 
 ### Adding a database connection
 
-Now you’ll see a list of your databases. To connect another database to Metabase, click **Add database**. Metabase currently supports the following types of databases:
+Now you’ll see a list of your databases. To connect another database to Metabase, click **Add database**. 
+
+#### Officially supported databases
 
 - Amazon Redshift
+- [BigQuery](databases/bigquery.md)
 - Druid
 - Google Analytics
```

Comment:
It would be nice to say that we only support Google Analytics V3 and not V4) https://github.com/metabase/metabase/issues/12905